config-service-impl/src/test/java/org/hypertrace/config/service/store/ConfigDocumentTest.java

```diff
@@ -23,6 +26,15 @@ void convertToAndFromJson() throws IOException {
     assertEquals(configDocument, ConfigDocument.fromJson(configDocument.toJson()));
   }
 
+  @Test
+  void convertDocumentContainingNullValue() throws IOException {
+    long timestamp = System.currentTimeMillis();
+    Value nullValue = Value.newBuilder().setNullValue(NullValue.NULL_VALUE).build();
+    ConfigDocument configDocument = new ConfigDocument(RESOURCE_NAME, RESOURCE_NAMESPACE,
+        TENANT_ID, DEFAULT_CONTEXT, 15, "user1", nullValue, timestamp, timestamp);
+    assertEquals(configDocument, ConfigDocument.fromJson(configDocument.toJson()));
```

Comment:
This assertion fails without the above changes in `ConfigDocument` as in that case, the config document - `ConfigDocument.fromJson(configDocument.toJson())` would have config value as `null`.